rules-reviewed/jboss-eap/websphere/tests/websphere-mq.windup.test.xml

```diff
@@ -0,0 +1,500 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ruletest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:wu="http://windup.jboss.org/schema/jboss-ruleset" xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="websphere-mq-test-r1" xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <testDataPath>samples/WebSphereMQ</testDataPath>
+    <rulePath>../websphere-mq.windup.xml</rulePath>
+    <ruleset id="websphere-mq" xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+
+        <metadata>
+            <description>
+                Provides analysis of WebSphere JMS proprietary classes and constructs that may require individual attention when migrating to JBoss EAP 6+.
+            </description>
+            <dependencies>
+                <addon id="org.jboss.windup.rules,windup-rules-javaee,2.2.0.Final"/>
+                <addon id="org.jboss.windup.rules,windup-rules-java,2.2.0.Final"/>
+            </dependencies>
+            <sourceTechnology id="websphere" versionRange="(0,)"/>
```

Comment:
probably better just to remove the versionRange. We made it optional in the XSD.